src/app/containers/ATIAnalytics/params/frontpage/buildParams.test.js

```diff
@@ -0,0 +1,92 @@
+import { buildFrontPageATIParams, buildFrontPageATIUrl } from './buildParams';
+import {
+  getPublishedDatetime,
+  getCurrentTime,
+} from '../../../../lib/analyticsUtils';
+
+jest.mock('../../../../lib/analyticsUtils', () => {
+  const utils = jest.requireActual('../../../../lib/analyticsUtils');
+
+  return {
+    ...utils,
+    getPublishedDatetime: jest.fn(),
+    getCurrentTime: jest.fn(),
+  };
+});
+
+getCurrentTime.mockImplementation(() => '00:00:00');
+getPublishedDatetime.mockImplementation(() => 'getPublishedDatetime');
```

Comment:
not a big deal but it would be nice if the mock return value resembled actual data.

Reply:
Fixed!